src/components/StatusIndicator/__tests__/__snapshots__/StatusIndicator.spec.js.snap

```diff
@@ -43,40 +45,22 @@ exports[`StatusIndicator renders 1`] = `
       status="incomplete"
     />
     <Button
-      className="security--status-indicator__retry-button"
+      className="security--status-indicator__button"
       disabled={false}
-      key="security--status-indicator__retry-button"
       kind="ghost"
       largeText={null}
       loading={false}
+      renderIcon={
+        Object {
+          "$$typeof": Symbol(react.forward_ref),
+          "render": [Function],
+        }
+      }
       tabIndex={0}
       type="button"
     >
-      <span
-        className="security--status-indicator__step__icon-wrapper"
-      >
-        <Icon
-          className="security--status-indicator__retry-button__icon"
```

Comment:
Can we still apply this class to the rendered Icon? As in, pass in the icon as a component with this class applied? Just in case someone is relying on the class name for a test or something like that. Same goes for the wrapper.